Jenkinsfile.github

```diff
@@ -66,6 +68,9 @@ pipeline {
                 script {
                     sh "python3 -m pip install setuptools_scm[toml]"
                     version = sh(returnStdout: true, script: "python3 -m setuptools_scm | tr -s '-' '~' | tr -s '+' '_' | sed 's/^v//'").trim()
+
+                    // lock .git for writing - otherwise getting "content has changed" error while preparing tar.bz2 for RPM
+                    sh "chmod -R a-w .git"
```

Comment:
I see.  There is a line in the makefile tar command to address the error, too..not sure if it is still needed there or not

Reply:
That's for pulling in git information so `python -m setuptools_scm` works in the RPM build.
I needed this because without it the RPM builds were failing during parallel 3.9 and 3.10 builds. Now.. I did remove the 3.9 build so it doesn't really matter.. but if we add aarch64 we'll have parallel builds again which'll run into the problem.